packages/tracing/src/browser/web-vitals/getCLS.ts

```diff
@@ -34,31 +34,27 @@ export interface LayoutShiftAttribution {
   currentRect: DOMRectReadOnly;
 }
 
-export const getCLS = (onReport: ReportHandler, reportAllChanges = false): void => {
+export const getCLS = (onReport: ReportHandler, reportAllChanges?: boolean): void => {
   const metric = initMetric('CLS', 0);
-
   let report: ReturnType<typeof bindReporter>;
 
   const entryHandler = (entry: LayoutShift): void => {
-    // Only count layout shifts without recent user input.
     if (!entry.hadRecentInput) {
       (metric.value as number) += entry.value;
       metric.entries.push(entry);
-      report();
+      if (report) {
+        report();
+      }
     }
   };
 
   const po = observe('layout-shift', entryHandler as PerformanceEntryHandler);
   if (po) {
-    report = bindReporter(onReport, metric, po, reportAllChanges);
+    report = bindReporter(onReport, metric, reportAllChanges);
 
-    onHidden(({ isUnloading }) => {
+    onHidden(() => {
       po.takeRecords().map(entryHandler as PerformanceEntryHandler);
-
-      if (isUnloading) {
-        metric.isFinal = true;
-      }
-      report();
+      report(true);
```

Comment:
I'd prefer `report({ force: true })` for readability, but I'm fine leaving it like this.

Reply:
Agree, but I'm matching the API from vendored version here.